data.json

```diff
@@ -350,6 +350,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories"]
   },
+  {
+    "name": "Appwrite",
+    "difficulty": "medium",
+    "description": "Make pull requests to issues tagged with \"Hacktoberfest\" or \"help wanted\" label in appwrite repo during October and get some stickers and badges.",
```

Comment:
```suggestion
    "description": "Make pull requests to issues tagged <code>Hacktoberfest<code> or <code>help wanted</code> in the <a href="https://github.com/appwrite/appwrite">appwrite repo</a> during October to get some stickers and badges.",
```

Reply:
@vikaspotluri123 The hyperlink is making the build fail

Reply:
It's because I forgot to escape the quotes 🙃

Reply:
AH xD